Trappist/src/Promact.Trappist.Repository/Tests/TestsRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Models.Test;
+using System.Linq;
+namespace Promact.Trappist.Repository.Tests
+{
+    public class TestsRepository : ITestsRepository
+    {
+        // List<Test> t_list = new List<Test>();
+        private readonly TrappistDbContext _dbContext;
+        public TestsRepository(TrappistDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+
+        /// <summary>
+        /// service for new test
+        /// </summary>
+        /// <param name="t1"></param>
```

Comment:
parameter name shoudl be proper and description shuold be there. what this parameter will do.